example/stringWrite/stringWrite.cpp

```diff
@@ -0,0 +1,41 @@
+#incldue <iostream>
+#incldue <sstream>
+#include <json/json.h>
+/*
+write a Value object to a string
+>g++ stringWrite.cpp -ljsoncpp -std=c++11 -o stringWrite
+>./stringWrite
+
+{
+    "action" : "run",
+    "data" :
+    {
+        "number" : 11
+    }
+}
+
+*/
+
+using namespace std ;
```

Comment:
Run clang format, and issues here like extra space before the semicolons will disappear.

Also, I would generally prefer no "using namespace std" declarations, even for example code.